ui/src/app/views/workspace-edit/component.ts

```diff
@@ -56,9 +57,25 @@ export class WorkspaceEditComponent implements OnInit {
       } else {
         this.buttonClicked = true;
         this.valueNotEntered = false;
-        this.errorHandlingService.retryApi(this.workspacesService.createWorkspace(this.workspace))
-            .subscribe(cohorts => this.router.navigate(['../..'], {relativeTo : this.route}));
+        this.errorHandlingService.retryApi(
+          this.workspacesService.createWorkspace(this.workspace))
+            .subscribe(
+              (workspace) => {
```

Comment:
I guess `cohorts` before was a typo (which we didn't use anyway)?

Reply:
It was (I think a copy/paste remnant.) I'm actually going to see if I can subscribe with no arguments, even if the call returns something.